cli/tests/e2e/snapshots/test_metrics/test_metrics_payload/pro_flag0/metrics-payload.json

```diff
@@ -1,11 +1,11 @@
 {
   "anonymous_user_id": "11111111-1111-1111-1111-111111111111",
   "environment": {
-    "ci": null,
+    "ci": "None",
     "configNamesHash": "d3e03c9938537c0a0668cec7204f37f1e9da87d03e18e8cc444dd7454b36c4d5",
     "integrationName": "funkyintegration",
     "isAuthenticated": false,
-    "projectHash": null,
+    "projectHash": "None",
```

Comment:
This seems incorrect.
@mjambon why we got that? we need nullable?
Or maybe it's because clara didn't use the latest version of atdpy when updating the files?

Reply:
cc @dinosaure